exercises/pangram/examples/success-text/src/Pangram.hs

```diff
@@ -0,0 +1,13 @@
+module Pangram (isPangram) where
+
+import qualified Data.Set as S
+import           Data.Set (Set)
+
+import qualified Data.Text as T
+import           Data.Text (Text)
+
+isPangram :: Text -> Bool
+isPangram = S.null . T.foldl' (flip S.delete) alphabet . T.toLower
```

Comment:
I've gone with a strict fold instead of lazy termination; while a list-based approach might terminate early by using `S.any null . L.scanl (flip S.delete)`, lazy `Text` is [generally discouraged](https://haskell-lang.org/tutorial/string-types) in favor of *conduits* and *pipes*.